### PR TITLE
display jobs without group if groupid==0

### PR DESCRIPTION
**Pull request by lnussel**
also clone job group

### CODE REVIEW

File: lib/OpenQA/Scheduler.pm [hunk 1]
```diff
@@ -184,6 +184,10 @@ sub job_create {
         delete $settings{NAME};
     }
 
+    if ($settings{_GROUP}) {
+        $new_job_args{'group'} = {'name' => delete $settings{_GROUP}};
```
Comment (coolo) [inline lib/OpenQA/Scheduler.pm]: please no useless quotes for hash keys.

File: lib/OpenQA/Scheduler.pm [hunk 2]
```diff
@@ -331,19 +335,29 @@ sub query_jobs {
...
+                });
+        }
+        else {
+            push(
+                @conds,
+                {
+                    'me.group_id' => $args{groupid},
+                });
+        }
     }
```
Comment (coolo) [inline lib/OpenQA/Scheduler.pm]: I think this whole construct can be written shorter as 

'me.group_id' => $args{groupid} || undef;

File: t/ui/02-list-group.t [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+#
+# List with no parameters
+#
+is($driver->get_title(), "openQA", "on main page");
+my $baseurl = $driver->get_current_url();
+
+#
+# Test the legacy redirection
+#
+is(1, $driver->get($baseurl . "tests"), "/tests");
```
Comment (aaannz) [inline t/ui/02-list-group.t]: Is there some reason to do this like that? If ok(..) is not suitable, at least please put expected value as second param. Else when the test fail you get i.e. expected undef, but got 1.

Comment (lnussel) [inline t/ui/02-list-group.t]: you are right, c&p from other test :)

**Pull request merged**